src/components/BookItem.jsx

```diff
@@ -2,23 +2,43 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/booksSlice';
+import { CircularProgressBar } from './Icons';
 
 const BookItem = ({ book }) => {
+  const randomPercentage = Math.floor(Math.random() * 101);
   const dispatch = useDispatch();
   return (
-    <>
-      <span>
-        <strong>Title:</strong>
-        {book.title}
-        ,
-        <strong>Author:</strong>
-        {book.author}
-        ,
-        <strong>Category:</strong>
-        {book.category}
-      </span>
-      <button type="button" onClick={() => dispatch(deleteBook(book))}>Delete</button>
-    </>
+    <div className="single-book-container">
+      <div className="single-book">
+        <div className="book-info">
+          <span className="book-category">{book.category}</span>
+          <span className="book-title">{book.title}</span>
+          <span className="book-author">{book.author}</span>
+        </div>
+        <div className="btn-container">
+          <button type="button">Comments</button>
+          <span>|</span>
+          <button type="button" onClick={() => dispatch(deleteBook(book))}>Remove</button>
+          <span>|</span>
```

Comment:
- [ ] Your button is on point, matching the color as intended in the design! However, I'd like to draw your attention to the divider color, which doesn't quite align with the design's specifications. For a more cohesive look, please take a moment to update the divider color so that it matches the design accurately. This attention to detail will contribute to a polished and harmonious visual presentation.

Great job on the button color, and thank you for your efforts in ensuring consistency throughout your project! 🎨👍